tfjs-layers/src/model_save_test.ts

```diff
@@ -53,6 +53,14 @@ describeMathCPUAndGPU('LayersModel.save', () => {
     expect(handler.savedArtifacts.convertedBy).toEqual(null);
   });
 
+  it('Save model artifact with dateSaved key', async () => {
+    const model = tfl.sequential();
+    const handler = new IOHandlerForTest();
+    model.add(tfl.layers.dense({units: 1, inputShape: [3]}));
+    await model.save(handler);
+    expect(handler.savedArtifacts.dateSaved).not.toBeNull();
```

Comment:
Nit: Can we test for the value being date-like? This regex isn't perfectly specific, but it approximates what we expect.
```suggestion
    expect(handler.savedArtifacts.dateSaved).toMatch(/\d+-\d+\d+@\d+:\d+:\d+/);```